GOT-StudentVersion/GOT-StudentVersion/DetailViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  DetailViewController.swift
+//  GOT-StudentVersion
+//
+//  Created by Angela Garrovillas on 8/12/19.
+//  Copyright © 2019 C4Q . All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
+    
+    @IBOutlet weak var episodeImage: UIImageView!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var seasonLabel: UILabel!
+    @IBOutlet weak var episodeNumLabel: UILabel!
+    @IBOutlet weak var runtimeLabel: UILabel!
+    @IBOutlet weak var airtimeLabel: UILabel!
+    @IBOutlet weak var summaryText: UITextView!
+    //make constraints and stretch for labels
+    
+    var episode: GOTEpisode?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard let episode = episode else {
```

Comment:
This is good, but wrap this in a private func and then call it in a view controller, to make it more clean.